parsers/YUKONENERGY.py

```diff
@@ -40,80 +34,85 @@ def fetch_production(
     logger: Logger = getLogger(__name__),
 ) -> list[dict[str, Any]]:
     """
-    Requests the last known production mix (in MW) of a given region.
+    Thermal is a mix of diesel and LNG, thermal is therefore set to unknown.
+    https://yukonenergy.ca/energy-in-yukon/projects-facilities/
+    """
 
-    We are using Yukon Energy's data from
-    http://www.yukonenergy.ca/energy-in-yukon/electricity-101/current-energy-consumption
+    if zone_key != ZONE_KEY:
+        raise ParserException("CA_YT.py", "Cannot parse zone '{zone_key}'", zone_key)
+    if target_datetime:
+        raise ParserException("CA_YT.py", "Unable to fetch historical data", zone_key)
 
-    Generation in Yukon is done with hydro, diesel oil, and LNG.
+    session = session or Session()
 
-    There are two companies, Yukon Energy and ATCO aka Yukon Electric aka YECL.
-    Yukon Energy does most of the generation and feeds into Yukon's grid. ATCO
-    does operations, billing, and generation in some of the off-grid
-    communities.
+    response = session.get(URL)
+    html = response.text
+    soup = BeautifulSoup(html, "html.parser")
 
-    See schema of the grid at http://www.atcoelectricyukon.com/About-Us/
+    # Extract JavaScript data
+    match_past_day = re.search(
+        r"var rows_chart_past_day = parseDates\((\[\[.*?\]\])\);", html, re.DOTALL
+    )
+    match_current = re.search(
+        r"var rows_chart_current = parseDates\((\[\[.*?\]\])\);", html, re.DOTALL
+    )
+    if not match_past_day or not match_current:
+        raise ParserException("CA_YT.py", "Cannot find data", zone_key)
```

Comment:
Probably should be the actual filename? 

```suggestion
        raise ParserException("YUKONENERGY.py", "Cannot find data", zone_key)
```

Also a couple other spots in this file. I assume this was missed when the file was renamed.

Reply:
Yes, the filenames should be changed! Some of the documenting comments were outdated, but we can keep some of the not outdated ones